requirements.txt

```diff
@@ -82,7 +82,7 @@ pluggy==1.0.0
 prometheus-client==0.14.1
 prompt-toolkit==3.0.30
 proto-plus==1.22.0
-protobuf==3.20.1
```

Comment:
I don't know if this will work with our current Google API version. Can you revert these `requirements` changes?